src/components/EmptyListMessage.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components/macro';
+import iconFeather from '../assets/icon_feather.png';
+import { useNavigate } from 'react-router-dom';
+
+export default function EmptyListMessage() {
+  const navigate = useNavigate();
+
+  function handleClick(e) {
+    navigate('../form');
+  }
+
+  return (
+    <StyledMessage onClick={handleClick}>
+      You don't have any notes yet! What's on your mind? Write it down right here...
+      <StyledIcon src={iconFeather} alt="feather" />
+    </StyledMessage>
```

Comment:
Is it obvious that the user has to click on the message? You should either use a button element or add `role="button"` to your `div` for accessibility reasons.